### PR TITLE
Enable PRs in sonarqube

### PR DESCRIPTION
**Pull request by chenson42**
### Summary
Enable PRs in sonarqube

### CODE REVIEW

File: openpos-assemble/Jenkinsfile [hunk 1]
```diff
@@ -15,23 +14,39 @@ pipeline {
                   -PbuildName="${JOB_NAME}" \
                   -PgitBranch="${BRANCH_NAME}" \
                   -PgitHash=${GIT_COMMIT} \
+                   clean build"""}
```
Comment (jchirico8) [inline openpos-assemble/Jenkinsfile:17]: Would it make sense to split clean and build? I don't know if gradle would do some extra junk if you tried clean on its own

Comment (chenson42) [reply openpos-assemble/Jenkinsfile]: It is not going to hurt anything to break it out.  Done.

**Comment by jchirico8** [comment]
Is this expected?
image

**Comment by chenson42** [comment]
> Is this expected?
> image

bitmoji

**Pull request merged**